control-spec.txt

```diff
@@ -579,6 +579,10 @@
       microdescriptors instead.  If microdescriptors are enabled, you'll
       need to use "md" instead.)
 
+    "md/all" -- all known microdescriptors for entire Tor network.
```

Comment:
Perhaps we should point out that the microdescriptors are joined by newlines.

Reply:
Wrote one more sentence in 0434d8446caf95f98784dfe647688e91dfce30e8.